### DISCUSSION TITLE
How can I zoom out from the camera?

### DISCUSSION THREAD

**Discussion by mervedemirel** [google/model-viewer#3291] [Q&A]
Hello, 

My 3D model is very close to camera and it is big. How can I zoom out? Is it possible?

Thanks.

**Comment by elalish** [comment]
Can you give a link so we can see what the problem is?

**Reply by mervedemirel** [reply]
I just want to zoom out a bit and make it smaller. Here is the link: mervedemirel.github.io

**Reply by mervedemirel** [reply]
Thanks, I resolved the problem. I used field-of-view to zoom out.
